### DISCUSSION TITLE
Headless style operation?

### DISCUSSION THREAD

**Discussion by projection-org** [artisan-roaster-scope/artisan#1551] [Ideas]
please delete, I used the wrong account.

**Comment by MAKOMO** [comment]
Artisan is monolithic and will most likely never be split into a backend and front end. However, it connects to "backends" like MODBUS, S7 or WebSocket server to gather data and to send control commands too. So Artisan IS a front end to those PLCs delivering the data and controlling the machine. A simple Arduino or RPi should be enough to control the actors of the machine, read the sensors and communicate with Artisan.

**Comment by projection-org** [comment]
please delete, I used the wrong account

**Comment by projection-org** [comment]
please delete, I used the wrong account
